### PR TITLE
[Sprint Maintain] Fix Validate In Screen Sign Up

### PR DESCRIPTION
**Pull request by tuanlvt-1117**
Fix Validate In Screen Sign Up
Chuyển từ toast thành errorTextInputLayout và validate theo các trường hợp : 
+ Email , mật khẩu, xác nhận mật khẩu trống
+ Email không đúng định dạng
+ Mật khẩu ít hơn 6 ký tự
+ Mật khẩu xác nhận không khớp với mật khẩu

### CODE REVIEW

File: app/src/main/java/com/framgia/englishconversation/utils/binding/BindingUtils.java [hunk 1]
```diff
@@ -147,4 +150,35 @@ public void onClick(View v) {
...
+        }
+        editText.addTextChangedListener(new TextWatcher() {
+            @Override
+            public void beforeTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+                //No-op
+            }
+
+            @Override
+            public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+                if (charSequence.length() < 1) {
```
Comment (fs-sournary) [inline app/src/main/java/com/framgia/englishconversation/utils/binding/BindingUtils.java]: hard code

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -88,4 +88,8 @@
     <string name="msg_get_user_info_failed"> .Please try to login again!</string>
     <string name="description_back_to_timeline">Press to back to timeline</string>
     <string name="description_vote_for_post">Vote for this post</string>
+    <string name="is_empty">Empty, Please enter the data!</string>
```
Comment (DoanVanToan) [inline app/src/main/res/values/strings.xml:91]: FATAL: [MissingTranslation] "`is_empty`" is not translated in "de" (German), "fr" (French), "ru" (Russian)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -88,4 +88,8 @@
...
+    <string name="invalid_email_format">Invalid email format!</string>
```
Comment (DoanVanToan) [inline app/src/main/res/values/strings.xml:92]: FATAL: [MissingTranslation] "`invalid_email_format`" is not translated in "de" (German), "fr" (French), "ru" (Russian)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -88,4 +88,8 @@
...
+    <string name="least_6_characters">Please enter at least 6 characters!</string>
```
Comment (DoanVanToan) [inline app/src/main/res/values/strings.xml:93]: FATAL: [MissingTranslation] "`least_6_characters`" is not translated in "de" (German), "fr" (French), "ru" (Russian)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -88,4 +88,8 @@
...
+    <string name="confirm_password_does_not_match">Confirm password does not match!</string>
```
Comment (DoanVanToan) [inline app/src/main/res/values/strings.xml:94]: FATAL: [MissingTranslation] "`confirm_password_does_not_match`" is not translated in "de" (German), "fr" (French), "ru" (Russian)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

**Review by TheVinhLuong — APPROVED** [review]

**Review by anhpham2403 — APPROVED** [review]

**Pull request merged**